packages/calling/src/CallingClient/calling/call.test.ts

```diff
@@ -854,6 +888,73 @@ describe('State Machine handler tests', () => {
     expect(call['callStateMachine'].state.value).toBe('S_SEND_CALL_DISCONNECT');
   });
 
+  it('outgoing call with connect after setup, media established before connect ', async () => {
```

Comment:
Message is not very clear, can we make it more readable?

Reply:
I see it. I've made this heading clearer now.
`outgoing call where we receive connect directly after setup. Media established before connect. test call and media state changes`